app/presenters/ministers_index_presenter.rb

```diff
@@ -14,4 +14,36 @@ def is_during_reshuffle?
   def reshuffle_messaging
     @ministers_index.content_item.details.dig("reshuffle", "message")
   end
+
+  def cabinet_ministers
+    @ministers_index.content_item.content_item_data.dig("links", "ordered_cabinet_ministers").map do |minister_data|
+      Minister.new(minister_data)
+    end
+  end
+
+  class Minister
```

Comment:
I'm not a huge fan of nested classes like this. Any chance we could change it to use a struct? 

https://github.com/alphagov/government-frontend/blob/main/app/presenters/field_of_operation_presenter.rb#L4

Reply:
I took inspiration from the [embassies index presenter](https://github.com/alphagov/collections/pull/3227/files#diff-09fcc2fb32a88072363c121b73ec077832ec758f5cddcf75d9f104e61c455907R20), which is in the same repo, aiming for a consistent style.